azalea/src/nearest_entity.rs

```diff
@@ -0,0 +1,181 @@
+use azalea_entity::Position;
+use azalea_world::{InstanceName, MinecraftEntityId};
+use bevy_ecs::{
+    prelude::Entity,
+    query::{ReadOnlyWorldQuery, With},
+    system::{Query, SystemParam},
+};
+
+/// This system parameter can be used as a shorthand for quickly finding an
+/// entity, (or several) close to a given position.
+///
+/// This system parameter allows for additional filtering of entities based off
+/// of ECS marker components, such as `With<>`, `Without<>`, or `Added<>`, etc.
+/// All functions used by this system parameter instance will respect the
+/// applied filter.
+///
+/// ```no_run
```

Comment:
no_run isn't necessary here since it's defining a function and the function isn't called.

Reply:
Turns out I need the `no_run` tag because of the imports. Otherwise the compiler complains about the missing imports used in the example.